desktop-core/src/main/java/com/stnetix/ariaddna/desktopcore/fileWatchigService/Consumer.java

```diff
@@ -0,0 +1,155 @@
+package com.stnetix.ariaddna.desktopcore.fileWatchigService;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.WatchEvent;
+import java.util.List;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
+
+import static java.nio.file.StandardWatchEventKinds.*;
+
+/**
+ * Class for transform native event to FileSystemWatchEvent
+ * it is necessary to receive RENAME and MOVE events
+ * because native events is only CREATE, DELETE and MODIFY
+ * <p>
+ * Depending on the file system RENAME and MOVE events is DELETE+RENAME native events
```

Comment:
Please make fixme tag that works on Windows NTFS and not testing on other OS and file system